### ISSUE TITLE
Adjust permissions on Metadata metrics

### ISSUE THREAD

**Issue by NicoledeGreef** [bcgov/MFIN-Data-Catalogue#574]
> As discussed, let's remove access to "Metadata metrics" report from any roles that are not **Catalogue admin** or **Catalogue information management specialist**.
> 

 _Originally posted by @NicoledeGreef in [#525](https://github.com/bcgov/MFIN-Data-Catalogue/issues/525#issuecomment-2297179117)_

- [ ] Modify the app so that only users with either the *Catalogue admin** or **Catalogue information management specialist** roles can see/access the "Metadata metrics" tab from their Dashboard
- [ ] Adjust documentation so that it is clear that only users with the **Admin** or **Info management specialist** roles have access to the "Metadata metrics" tab.

**Comment by chrislaick** [comment]
Just updating the notes on this ticket after speaking with Nicole. For **Catalogue admin** and **Catalogue information management specialist** roles they will still see the Metadata metrics report page. For all other users, they'll see the "Report builder" page under the Reports parent tab.

**Comment by chrislaick** [comment]
This has been deployed to DEV and TEST and is ready for QA. I managed to make the access modifications without affecting the URL structure. Here is a screenshot of the `/reports` page for user roles that are not Catalogue admin or Catalogue information management specialist:

![Image](https://github.com/user-attachments/assets/b3fe63ff-0c86-4e28-b85d-81715d9a0ae6)

**Comment by NicoledeGreef** [comment]
Reviewed in Test using drupal account with no admin role as well as regular IDIR account with admin role.

**Comment by chrislaick** [comment]
Deployed to PROD.

**Comment by david-fong-bc** [comment]
I can update [the documentation](https://cat.data.fin.gov.bc.ca/documentation/reports#metadata-metrics). I just want to make sure I understand what the expected behaviour is. Is the change that we ended up going with to hide the pie charts from users that aren't Data Catalogue Admin or Information Specialist?

**Comment by chrislaick** [comment]
@david-fong-bc That is correct. The pie charts are hidden for users that are _not_ Data Catalogue administrators or Information Specialists. We can close this ticket as it's Done.

**Comment by david-fong-bc** [comment]
Sounds good. @NicoledeGreef, I made the following addition to [the documentation](https://cat.data.fin.gov.bc.ca/documentation/reports#metadata-metrics):

> Note that the pie charts are only shown to users with the Data Catalogue Administrator or Information Specialist role.

Let me know if you'd suggest any adjustments. Otherwise, I think as Chris suggested, this issue can be closed?
